widgets/contact/contact.php

```diff
@@ -343,6 +343,72 @@ function get_widget_form(){
 						),
 					),
 
+					'fields' => array(
+						'type'   => 'section',
+						'label'  => __( 'Fields', 'so-widgets-bundle' ),
+						'fields' => array(
+							'font' => array(
+								'type'    => 'font',
```

Comment:
This needs to be added to the array returned by `get_google_font_fields()` to get the font imported properly. I.e.
```
	function get_google_font_fields( $instance ) {
		return array(
			$instance['design']['labels']['font'],
			$instance['design']['fields']['font'],
		);
	}
```
 At some point we still want to try simplify the process of adding and using a font field, so that this won't be necessary.